app/src/main/res/values-nl/strings.xml

```diff
@@ -0,0 +1,13 @@
+<resources>
+    <string name="app_name">Android versie</string>
+    <string name="codename">Code name: %s\n</string>
+    <string name="version">Versie: %s (API %d)\n</string>
+    <string name="based_on">ROM gebaseerd op: %s\n</string>
+    <string name="preview">Preview-versie van %s (API %d)\n</string>
+    <string name="patch">Patch niveau: %s\n</string>
+    <string name="made_by">Auteur</string>
+    <string name="license">Licentie</string>
+    <string name="android_trademark">Android is a trademark of Google LLC.</string>
```

Comment:
This string doesn't seem to be translated. Can you translate or remove it?

Reply:
Maybe some words are the same as in English and it looks not translated, but is is. Example: License in English, Licentie in Dutch ...

Reply:
I was only commenting on `android_trademark`, which is translated now. Thanks!

Reply:
Oke, You are welcome